pkg/cedar/core/client.go

```diff
@@ -154,3 +158,123 @@ func (c *Client) GetSystem(ctx context.Context, id string) (*models.CedarSystem,
 		SystemMaintainerOrgComp: responseArray[0].SystemMaintainerOrgComp,
 	}, nil
 }
+
+// GetDeploymentsOptionalParams represents the optional parameters that can be used to filter deployments
+// when searching through the CEDAR API
+// TODO - is this how we want to handle multiple optional parameters?
```

Comment:
I didn't see an existing pattern for how to handle this, so I went with bundling the optional parameters into a struct. Does this seem like a good way to handle this?

Note to self: if team agrees, remove TODO note

Reply:
This seems right to me. Keeps us from having a function definition with `null, null, null` (or I guess `"","",""` in this case?)

Reply:
TODO note removed in https://github.com/CMSgov/easi-app/pull/1406/commits/9e8e5e56fa8caf8885a04ec4469ab7aaff7119b1.